src/components/SidebarTabs/SettingsSidebarTab.vue

```diff
@@ -59,6 +59,27 @@
 				{{ t('forms', 'Show expiration date on form') }}
 			</NcCheckboxRadioSwitch>
 		</div>
+		<NcCheckboxRadioSwitch :checked="hasCustomSubmissionMessage"
+			type="switch"
+			@update:checked="onUpdateShowSubmissionMessageInput">
+			{{ t('forms', 'Custom submission message') }}
+		</NcCheckboxRadioSwitch>
+		<div v-show="hasCustomSubmissionMessage"
+			class="settings-div--indent submission-message"
+			:tabindex="editMessage ? undefined : '0'"
+			@focus="editMessage = true">
+			<textarea v-if="editMessage || !form.submissionMessage"
+				v-click-outside="() => { editMessage = false }"
+				:aria-label="t('forms', 'Custom submission message')"
+				:value="form.submissionMessage"
+				:maxlength="maxStringLengths.submissionMessage"
+				:placeholder="t('forms', 'Message to show after an user submitted the form (formatting using Markdown is supported)')"
```

Comment:
```suggestion
				:placeholder="t('forms', 'Message to show after a user submitted the form (formatting using Markdown is supported)')"
```

It's `a user` as it's pronounced `ˈjuːzɐ`